episodes/infrastructure.md

```diff
@@ -246,6 +248,12 @@ We will revisit the `life_cycle` and `carpentry` fields in `config.yaml` later i
 
 :::
 
+#### `.Rproj` file
+
+The repository also contains a file called `FIXME.Rproj`, which should be renamed.
+You can call this file whatever you like,
+but the convention is for its name to match that of the lesson repository.
+
```

Comment:
```suggestion
#### `.Rproj` file

The repository also contains a file called `FIXME.Rproj`, which is the RStudio project file for your lesson with some options pre-set (and the recommended tool to edit and render your lesson locally on your machine). This file should be renamed.
You can call this file whatever you like,
but the convention is for its name to match that of the lesson repository.

```

Reply:
Feel like we need some explanation of what `.Rproj` file is.

Reply:
Good catch, thanks @anenadic. I'm going to trim your suggestion down a little and then merge.